### ISSUE TITLE
2.0.0 – Saving new chunk not possible

### ISSUE THREAD

**Issue by leonie12** [evolution-cms/evolution#936]
PHP 7.3:

When making new chunk and trying to save it I get following error:
``« Evo Parse Error »
SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'editor_name' cannot be null (SQL: insert into `site_htmlsnippets` (`name`, `description`, `snippet`, `locked`, `category`, `editor_type`, `editor_name`, `disabled`, `editedon`, `createdon`) values (test, testchunk, [+blablablub+], 0, 5, 1, , 0, 1545737891, 1545737891))``

```
Error information 

File | /core/vendor/illuminate/database/Connection.php
Line | 664


Backtrace

Illuminate\Support\Facades\Facade::__callStatic('handle', array $var2)manager/index.php on line 197
EvolutionCMS\ManagerTheme->handle(79)core/vendor/illuminate/support/Facades/Facade.php on line 223
Illuminate\View\View->render()core/src/ManagerTheme.php on line 388
Illuminate\View\View->renderContents()core/vendor/illuminate/view/View.php on line 90
Illuminate\View\View->getContents()core/vendor/illuminate/view/View.php on line 125
Illuminate\View\Engines\PhpEngine->get(string $var1, array $var2)core/vendor/illuminate/view/View.php on line 142
Illuminate\View\Engines\PhpEngine->evaluatePath(string $var1, array $var2)core/vendor/illuminate/view/Engines/PhpEngine.php on line 21
include(string $var1)core/vendor/illuminate/view/Engines/PhpEngine.php on line 43
include_once(string $var1)manager/views/page/79.php on line 4
Illuminate\Database\Eloquent\Model::__callStatic('create', array $var2)manager/processors/save_htmlsnippet.processor.php on line 53
Illuminate\Database\Eloquent\Model->__call('create', array $var2)core/vendor/illuminate/database/Eloquent/Model.php on line 1582
Illuminate\Database\Eloquent\Builder->create(array $var1)core/vendor/illuminate/database/Eloquent/Model.php on line 1570
tap(EvolutionCMS\Models\SiteHtmlsnippet $var1, Closure $var2)core/vendor/illuminate/database/Eloquent/Builder.php on line 756
Illuminate\Database\Eloquent\Builder->Illuminate\Database\Eloquent\{closure}(EvolutionCMS\Models\SiteHtmlsnippet $var1)core/vendor/illuminate/support/helpers.php on line 1041
Illuminate\Database\Eloquent\Model->save()core/vendor/illuminate/database/Eloquent/Builder.php on line 755
Illuminate\Database\Eloquent\Model->performInsert(Illuminate\Database\Eloquent\Builder $var1)core/vendor/illuminate/database/Eloquent/Model.php on line 615
Illuminate\Database\Eloquent\Model->insertAndSetId(Illuminate\Database\Eloquent\Builder $var1, array $var2)core/vendor/illuminate/database/Eloquent/Model.php on line 752
Illuminate\Database\Eloquent\Builder->__call('insertGetId', array $var2)core/vendor/illuminate/database/Eloquent/Model.php on line 787
Illuminate\Database\Query\Builder->insertGetId(array $var1, 'id')core/vendor/illuminate/database/Eloquent/Builder.php on line 1283
Illuminate\Database\Query\Processors\Processor->processInsertGetId(Illuminate\Database\Query\Builder $var1, string $var2, array $var3, 'id')core/vendor/illuminate/database/Query/Builder.php on line 2494
Illuminate\Database\Connection->insert(string $var1, array $var2)core/vendor/illuminate/database/Query/Processors/Processor.php on line 32
Illuminate\Database\Connection->statement(string $var1, array $var2)core/vendor/illuminate/database/Connection.php on line 411
Illuminate\Database\Connection->run(string $var1, array $var2, Closure $var3)core/vendor/illuminate/database/Connection.php on line 459
Illuminate\Database\Connection->runQueryCallback(string $var1, array $var2, Closure $var3)core/vendor/illuminate/database/Connection.php on line 624
```

This happens when the content of the chunk is as simple as `[+test+]`

**Comment by leonie12** [comment]
@AgelxNash: I am very interested to get to know why this bug is low priority. From a user perspective I see it as a heavy bug when it is impossible to make new chunks …

**Comment by dmi3yy** [comment]
Try create new chunk on 7.2 all ok

**Comment by dmi3yy** [comment]
http://take.ms/0OERv

**Comment by AgelxNash** [comment]
@leonie12 7.3.0 has some problems that I hope will be solved at php 7.3.1
Example: https://github.com/doctrine/doctrine2/issues/7531
Because of this problem, we can not run 2.0 on 7.3.0RC6 or later.
Before fixing the parent problem, we will have a low priority
